### ISSUE TITLE
sb-vision support

### ISSUE THREAD

**Issue by trickeydan** [j5api/j5#157]
Do we want to support `sb-vision`?

Depends on #47

**Comment by trickeydan** [comment]
Unsure if `sb-vision` is still maintained? @Adimote

**Comment by trickeydan** [comment]
In my opinion, this should go down as a `wontfix`. I can't see a situation where we're need to support `sb-vision` in the near to medium-term future.

**Comment by kierdavis** [comment]
I agree; particularly if we focus our effort towards libkoki support instead.
